### ISSUE TITLE
Resampling datetime if frequency isn't supported for ARIMA

### ISSUE THREAD

**Issue by ParthivNaresh** [alteryx/evalml#2593]
After #2592, the current implementation should skip inclusion of `ARIMA` if an incompatible frequency is detected.

This issue tracks resampling the data when it is passed to ARIMA to reach the level of granularity that is acceptable by `sktime`. This could involve rounding up or down in granularity i.e. previous frequency of 12 hours could be rounded up to 1 day with half the observations listed.

The actual implementation will require some design work and conversations on how data should be imputed if the frequency is being rounded down to a finer granularity, or consolidated if rounded up in granularity.

**Comment by tyler3991** [comment]
Closing, no longer relevant
